i18n/client.js

```diff
@@ -6,14 +6,16 @@ export default new NextI18Next({
   use: [LanguageDetector, initReactI18next],
   defaultLanguage: 'en',
   fallbackLng: 'en',
+  debug: false,
   otherLanguages: ['de'], // list all languages here
   detection: {
-    // check if language is cached in cookies, if not check local storage
-    order: ['cookie', 'localStorage'],
+    // check if language is cached in cookies, if not check local storage, 
+    // last retrieve from browser language
+    order: ['cookie', 'localStorage', 'navigator'],
```

Comment:
Added 'navigator' for automatic browser language detection here if cookie or storage are empty.